Model/Import/Category.php

```diff
@@ -1441,14 +1439,15 @@ public function getAttributeOptions(\Magento\Eav\Model\Entity\Attribute\Abstract
                     foreach (is_array($option['value']) ? $option['value'] : [$option] as $innerOption) {
                         if (strlen($innerOption['value'])) {
                             // skip ' -- Please Select -- ' option
-                            $options[$innerOption['value']] = (string)$innerOption[$index];
+                            $options[strtolower($innerOption['value'])] = (string)$innerOption[$index];
```

Comment:
According to ```vendor\magento\module-import-export\Model\Import\AbstractEntity.php:(Line: 660)``` this needs to be changed.
Had problems with ```display_mode``` attribute. It had ```PRODUCTS``` as a value